assets/texts/fr/amoDescription.html

```diff
@@ -0,0 +1,24 @@
+Ce module complémentaire (extension) pour Firefox vous donne un bon petit sélecteur d'émoticônes pour en trouver et en sélectionner. Il est copié dans votre presse-papier quand vous cliquez dessus.
+Cette application utilise des émoticônes à jour et vous laisse accèder à des émoticônes dans de différents styles que vous connaissez de différents appareils, sites et applications.
+Il est basé sur le superbe <a
```

Comment:
I guess there are a lot of spaces here/not needed line breaks.

Reply:
Oh, sorry; I thought that HTML ignored line breaks without `<br>` like in Markdown 😅